docs/api/v3.rst

```diff
@@ -552,34 +523,6 @@ This allows for documentation projects to share a search index and a namespace o
 but still be maintained independently.
 See :doc:`/subprojects` for more information.
 
-
-Subproject details
-++++++++++++++++++
```

Comment:
Looks like this shouldn't be deleted

Reply:
Hrm... I probably made a mistake when solving merge conflicts. Good catch!